.github/workflows/build.yml

```diff
@@ -24,37 +23,99 @@ jobs:
                     - '8.1'
                 symfony-version:
                     - '4.4.*'
-                    - '5.3.*'
                     - '5.4.*'
                     - '6.0.*'
                 exclude:
                     - php-version: '7.4'
                       symfony-version: '6.0.*'
         steps:
             - name: Checkout
-              uses: actions/checkout@v2
+              uses: actions/checkout@v3
 
             - name: Setup PHP
               uses: shivammathur/setup-php@v2
               with:
                   coverage: none
-                  ini-values: "memory_limit=-1"
+                  ini-values: "memory_limit=-1, zend.assertions=1"
```

Comment:
Just to be sure... this was not set in my local dev box, causing all tests to pass since `assert()` was not evaluated.